src/browser/api_protocol/api_handlers/system.js

```diff
@@ -41,6 +41,7 @@ function SystemApiHandler() {
         'get-el-ipc-config': getElIPCConfig,
         'get-environment-variable': { apiFunc: getEnvironmentVariable, apiPath: '.getEnvironmentVariable' },
         'get-host-specs': { apiFunc: getHostSpecs, apiPath: '.getHostSpecs' },
+        'get-min-log-level': getMinLogLevel,
```

Comment:
As mentioned in adapter PR, my feeling is that `getMaxLogLevel`/`setMaxLogLevel` might read with more meaning. As in "this is the _most_ detail we will log"... this means reversing the numbering of the levels. Am I the only one looking at it this way?